apps/website/src/core/component/link/link.presenter.tsx

```diff
@@ -1,13 +1,14 @@
 import NextLink from 'next/link';
 import type { ComponentPropsWithoutRef, FC } from 'react';
+import { twMerge } from '@/core/util/tw-merge.util';
 
-type LinkProps =
+export type LinkProps =
```

Comment:
[任意] `ComponentWithoutProps` で参照できるから別にいらなくね？🤔 と思ったり。。
むしろサードパーティー製のやつは `HogeProps` がないものもあるから  `ComponentWithoutProps` を使ってるコードと使ってないコードができてしまうなぁと


```suggestion
type LinkProps =
```

Reply:
`export`しないようにしといた